cmd/admin.go

```diff
@@ -145,3 +154,41 @@ func runCreateUser(c *cli.Context) error {
 	fmt.Printf("New user '%s' has been successfully created!\n", c.String("name"))
 	return nil
 }
+
+func runRepoSyncReleases(c *cli.Context) error {
+
+	setting.NewContext()
+	models.LoadConfigs()
+
+	setting.NewXORMLogService(false)
+	if err := models.SetEngine(); err != nil {
+		return fmt.Errorf("models.SetEngine: %v", err)
+	}
+
+	repos, count, err := models.SearchRepositoryByName(&models.SearchRepoOptions{})
+	if err != nil {
+		log.Fatal(4, "SearchRepositoryByName: %v", err)
+		return nil
```

Comment:
If we return `nil`, is the return status 0? I think that if we aren't able to even the load the repositories, then the exit status should be non-zero (e.g. if someone is running this command as part of a shell script).